docs/src/main/asciidoc/maven-tooling.adoc

```diff
@@ -456,6 +456,110 @@ If you have not used <<project-creation,project scaffolding>>, add the following
 <4> Enable the `native` package type. The build will therefore produce a native executable.
 <5> If you want to test your native executable with Integration Tests, add the following plugin configuration. Test names `*IT` and annotated `@NativeImageTest` will be run against the native executable. See the link:building-native-image[Native executable guide] for more info.
 
+[[multi-module-maven]]
+=== Multi-Module Maven project
+To set up a multi-module maven project we need to adjust the configuration given above slightly. Add or replace the following elements in your root `pom.xml` of the parent module:
+
+[source,xml,subs=attributes+]
+----
+<project>
+    [...]
+    <groupId>org.acme</groupId>
+    <artifactId>quarkus-multi-module-example</artifactId>
+    <packaging>pom</packaging> <1>
+    <version>${revision}</version> <2>
+    <properties>
+        <revision>1.0.0-SNAPSHOT</revision>
+        [...]
+    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency> <3>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom</artifactId>
+                <version>${quarkus.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    <build>
+        [...]
+        <pluginManagement>
+            <plugins>
+                <plugin> <4>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-maven-plugin</artifactId>
+                    <version>${quarkus.version}</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+    [...]
+    <modules> <5>
+        <module>webapp</module>
+        <module>domain</module>
+        <module>...</module>
+    </modules>
+</project>
+----
+
+<1> Set the packaging to `pom` as the parent module doesn't include any source code.
+<2> Set the version as a property to reuse the value in submodules.
+<3> Optionally use a BOM file to omit the version of the different Quarkus dependencies in your submodules.
+<4> Remove the `build` step from the plugin, we will add this to one of the submodules.
+<5> Set up the submodules
+
+Next create the `pom.xml` files for the submodules in their respective subfolders:
+[source,xml,subs=attributes+]
+----
+<project>
+    [...]
+    <parent> <1>
+        <groupId>org.acme</groupId>
+        <artifactId>quarkus-multi-module-example</artifactId>
+        <version>${revision}</version>
+    </parent>
+    <artifactId>webapp</artifactId> <2>
+    <build>
+        <plugins>
+            <plugin> <3>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>build</goal>
```

Comment:
So you suggest to build a quarkus application _per submodule_? This actually not really _required_.
You can do so but you can also have Modules completely without `quarkus-maven-plugin` (e.g. some "shared" libraries).

Reply:
Good catch. It is indeed not needed only needed for your "webapp" module. All other modules don't have to depend on Quarkus (that was the actual idea when splitting up my project). 

I noted this here https://github.com/quarkusio/quarkus/pull/9164/files/6fc309549a8fea8d6ee078d6cbcf59a9a71d48bc#diff-08ed80fdba6156d04b65ff7da6eac102R553 but we might have to make it more obvious.